frontend/src/components/UserHomepage.js

```diff
@@ -299,38 +290,25 @@ const UserHomepage = () => {
     ));
   };
 
-  // New function to navigate to home
-  const navigateToHome = () => {
-    navigate('/home');
-  };
-
-
   return (
     <div className="user-homepage-container">
-      <header className="user-homepage-header">
-        {/* Updated div to include onClick for navigation */}
-        <div className="busy-beaver-logo" onClick={navigateToHome} style={{ cursor: 'pointer' }}>
-          <img src={BusyBeaverNoBG} alt="Busy Beaver" />
-        </div>
-        {/* Rest of the component remains the same */}
-        <h1>User Homepage</h1>
-        <div className="user-homepage-buttons">
-          <input
-            type="text"
-            value={teamName}
-            onChange={(e) => setTeamName(e.target.value)}
-            placeholder="Enter new team name"
-            className={error ? "input-error" : ""}
-          />
-          <button onClick={handleAddTeam}>Add Team</button>
-          <button onClick={handleLogout}>Logout</button>
-          {error && <div className="error-message">{error}</div>}
-        </div>
-      </header>
+      <TopBar /> {/* Includes the TopBar component */}
       <div className="content-container">
         <aside className="team-list">
           <h2>Teams</h2>
           {renderTeams()}
+          {/* Add Team Section */}
+          <div className="add-team-section">
+            <input
```

Comment:
Are we thinking we'll hide these text boxes (here and add project) behind a button?

Reply:
Yep, I'll change those for the next feature branch. Are there specific MUI's you liked using for the project tasks page?

Reply:
I've been using Button, Typography, and TextField quite a bit